tests/integration/launch_container.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+docker_image=$1
+model_path=$2
+host_device=$3
+
+# start the docker container
+docker run \
+  -itd \
```

Comment:
Do we need to add runtime=nvidia for GPU? @lanking520 @frankfliu

Reply:
yes, or GPU will not be attached